includes/compat/subscriptions/class-wc-payment-gateway-wcpay-subscriptions-compat.php

```diff
@@ -152,7 +152,7 @@ public function scheduled_subscription_payment( $amount, $renewal_order ) {
 							'code'   => '<code>',
 						]
 					),
-					wc_price( $amount ),
+					wc_price( $amount, [ 'currency' => WC_Payments_Utils::get_order_original_currency( $renewal_order ) ] ),
```

Comment:
With reference to my another comment, we could use `$renewal_order->get_currency()` here (and similar calls in refund processing).

Reply:
Answered here https://github.com/Automattic/woocommerce-payments/pull/1227#discussion_r564995909.